057-marking-zk-kraft-clusters.md

```diff
@@ -0,0 +1,59 @@
+# Marking ZooKeeper and KRaft based clusters for KRaft enabled operators
```

Comment:
I did not notice it initially ... but mAybe we can call it something like "Allow running ZooKeeper- and KRaft-based clusters in parallel"? I do not think I would understand what it is about from the current title if I didn't know about it upfront.

Reply:
Yeah I was looking at a better title, I think your suggestion makes more sense.